docker/appserver/WebSphere/src/scripts/configureJvmProperties.py

```diff
@@ -21,10 +21,9 @@ def setSecurityProperty( name, value ):
 	return
 
 setJVMSystemProperty( 'log.dir', '${SERVER_LOG_ROOT}')
-setJVMSystemProperty( 'dtap.stage', 'TST')
+setJVMSystemProperty( 'dtap.stage', 'LOC')
```

Comment:
Is het nodig om de stage 'LOC' te hebben? De tests draaien toch ook met stage 'TST'?
M.a.w. wat is de reden voor deze aanpassing?

Reply:
Tomcat en JBOSS draaien ook als LOC, daarom heb ik dat gelijk getrokken, of is er een reden dat Websphere als TST moet draaien?

Reply:
Ik denk dat het beter is om de andere containers ook als TST te gaan laten draaien, omdat er dan https gebruikt wordt in plaats van http.

Reply:
Websphere staat nu weer op TST, andere containers ook naar TST om zetten zou ik buiten deze PR laten.